src/lib/dom.generated.d.ts

```diff
@@ -5464,6 +5464,10 @@ declare var HTMLLegendElement: {
 
 interface HTMLLinkElement extends HTMLElement, LinkStyle {
     /**
+     * Sets or retrieves the target attribute.
+     */
+    as: string;

```

Comment:
The spec you linked lists what are the valid keys, so I thought they could be enumerated here, but it's actually a freeform string (cookie-like)...